src/st/compatibility/interface-iwpml-page-builders-data-settings.php

```diff
@@ -44,5 +44,13 @@ public function prepare_data_for_saving( array $data );
 	 */
 	public function get_pb_name();
 
+	/**
+	 * @return void
+	 */
 	public function add_hooks();
-}
\ No newline at end of file
+
+	/**
+	 * @return bool
+	 */
+	public function should_copy_post_body();
```

Comment:
@adrianosferreira The problem with adding a new function is that all the classes that implement this now need to be updated.

I suggest that the call to `get_fields_to_copy` can also return `body`. Then the code can check for this and copy the body accordingly.

Reply:
@brucepearson I already created PR for Elementor and Beaver Builder. Do you think I should change it?

Reply:
@brucepearson what you said makes sense. I will change that.